CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Change log
 
+- [#498](https://github.com/mobilityhouse/ocpp/issues/498) Remove support for OCPP 2.0
```

Comment:
Can you group all breaking changes in their own section? And for this breaking change, you can link to https://github.com/mobilityhouse/ocpp/issues/498. That issue explains the breaking change and provides an upgrade path for users.